router/package_request_parts.go

```diff
@@ -253,7 +253,7 @@ func isFullSHASelector(selector string) bool {
 
 func isShortSHASelector(selector string) bool {
 	// If it isn't a 6 character short SHA return false, it might be semvar
-	return len(selector) == shortSHALength && strings.IndexByte(selector, dot) == -1 && strings.IndexByte(selector, hyphen) == -1
+	return len(selector) == minShortSHALength && strings.IndexByte(selector, dot) == -1 && strings.IndexByte(selector, hyphen) == -1
```

Comment:
This should be an inequality. Should be greater or equal to min short sha length and less than full sha length